package.json

```diff
@@ -108,6 +108,7 @@
     "@angular/pwa": "^12.0.5",
     "@angular/router": "^12.0.5",
     "@angular/service-worker": "^12.0.5",
+    "@lhci/cli": "^0.8.1",
```

Comment:
I see it also installed globally in action? Should it be also in our package.json? If so then I guess in devDependencies

Reply:
nope. Removing